daemon/proto/daemon.proto

```diff
@@ -7,7 +7,7 @@ service Daemon {
   rpc ListUsers (Empty) returns (ListUsersResponse) {}
   rpc ChangeUserPasswd (UpdatePasswdRequest) returns (UpdatePasswdResponse) {}
   rpc DestroyUser (DestroyUserRequest) returns (DestroyUserResponse) {}
-
+  rpc SetTeamSuspend (SetTeamSuspendRequest) returns (Empty) {}
```

Comment:
is this one used to suspend and resume at the same time ?

Reply:
Yeah, in suspend request it checks whether [suspend value](https://github.com/aau-network-security/haaukins/blob/a8abc3a3613b5b363c3b3a41ad0e88d37ccdd5e6/daemon/proto/daemon.proto#L39) set to true or false. If it is false which means resume called in back end.

Reply:
I see, so maybe its better to change the name of that function

Reply:
What would be your proposal for the name, then :) ?

Reply:
this one is okay now